geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/SimpleFeatureValidator.scala

```diff
@@ -199,11 +199,12 @@ object SimpleFeatureValidator extends LazyLogging {
     * @param geom geom index
     */
   private class Z2Validator(geom: Int) extends Validator {
+    private val envelope = wholeWorldEnvelope
     override def validate(sf: SimpleFeature): String = {
       val g = sf.getAttribute(geom).asInstanceOf[Geometry]
       if (g == null) {
         Errors.GeomNull
-      } else if (!wholeWorldEnvelope.contains(g.getEnvelopeInternal)) {
+      } else if (!envelope.contains(g.getEnvelopeInternal)) {
```

Comment:
Why did you pull envelope into this class?

Reply:
wholeWorldEnvelope is actually a `def`, since envelope is mutable